llvm/test/Transforms/GVNHoist/hoist-merge-geps.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: opt -S -passes=gvn-hoist < %s | FileCheck %s 
+
+; Function Attrs: noinline nounwind uwtable
```

Comment:
Minor nits, we can delete the `; Function Attrs` lines and the attribute references (`#0` and `#1`).

Reply:
Thanks for your comments. I fixed these minor issues, and now the test would be better.